### PR TITLE
Use the window.imagePath global for the base image path in examples if present.

### PR DESCRIPTION
**Pull request by bleathem**
This PR allows that patternfly.org site to override the image path used in some examples by specifying the IMAGE_PATH global variable.

### CODE REVIEW

File: src/card/aggregate-status/aggregate-status-card.component.js [hunk 1]
```diff
@@ -68,7 +68,8 @@
  </file>
 
  <file name="script.js">
-   angular.module( 'patternfly.card' ).controller( 'CardDemoCtrl', function( $scope ) {
+   angular.module( 'patternfly.card' ).controller( 'CardDemoCtrl', function( $scope, $window ) {
+    var imagePath = $window.IMAGE_PATH || "img";
```
Comment (jeff-phillips-18) [inline src/card/aggregate-status/aggregate-status-card.component.js]: we should be consistent and use /img

Comment (bleathem) [reply src/card/aggregate-status/aggregate-status-card.component.js]: I was trying to preserve what was previously present.  Would you say `img` is expected to be correct over '/img'

File: src/navigation/application-launcher.component.js [hunk 2]
```diff
@@ -77,8 +77,9 @@
    </div>
  </file>
  <file name="script.js">
-   angular.module('patternfly.navigation').controller('applicationLauncherController', ['$scope',
-     function ($scope) {
+   angular.module('patternfly.navigation').controller('applicationLauncherController', ['$scope', '$window',
+     function ($scope, $window) {
+       $scope.imagePath = $window.IMAGE_PATH || "img";
```
Comment (jeff-phillips-18) [inline src/navigation/application-launcher.component.js]: and again...

Comment (bleathem) [reply src/navigation/application-launcher.component.js]: ...again, I was preserving what was previously present.  Happy to converge on a single path with this PR though.

File: src/card/aggregate-status/aggregate-status-card.component.js (hunk 1 above)
Comment (jeff-phillips-18) [reply src/card/aggregate-status/aggregate-status-card.component.js]: yeah, I would say 'img' is probably better, I'd just rather see consistency.

Comment (bleathem) [reply src/card/aggregate-status/aggregate-status-card.component.js]: Done.  Standardised on `img`.

**Pull request merged**